packages/resolver-svimg/src/index.ts

```diff
@@ -0,0 +1,44 @@
+import { Queue, generateComponentAttributes } from 'svimg/dist/process.js';
+
+const SVIMG_TYPE = `
+type Svimg {
+  srcset: String
+  srcsetwebp: String
+  srcsetavif: String
+  placeholder: String
+  aspectratio: Float
+}`;
+
+const queue = new Queue();
+
+// copied from https://github.com/xiphux/svimg/blob/master/src/component/generate-component-attributes.ts#L9
+interface Config {
+  src: string;
+  queue?: Queue;
+  inputDir: string;
+  outputDir: string;
+  publicPath?: string;
+  webp?: boolean;
+  avif?: boolean;
+  widths?: number[];
+  quality?: number;
+  skipGeneration?: boolean;
+  skipPlaceholder?: boolean;
+}
```

Comment:
To stay lock-step with any changes that `svimg` introduces to config, we can replace this with:
```suggestion
type Config = Parameters<typeof generateComponentAttributes>[0];
```

Reply:
That's amazing! I didn't know you could do that